test/regression/expected/temporary_tables.out

```diff
@@ -407,10 +407,10 @@ SELECT count(*) FROM td;
 
 TRUNCATE TABLE tc;
 EXPLAIN VERBOSE INSERT INTO tc(c) SELECT md5('ta');
-                          QUERY PLAN                          
---------------------------------------------------------------
+                         QUERY PLAN                         
+------------------------------------------------------------
  Custom Scan (DuckDBScan)  (cost=0.00..0.00 rows=0 width=0)
-   Output: duckdb_scan.explain_key, duckdb_scan.explain_value
+   Output: duckdb_scan."Count"
```

Comment:
This is nice additional benefit of this change. The VERBOSE flag now actually lists the columns that the scan returns. For INSERT statements this isn't actually true though, because in postgres those don't return a column at all.